components/topic-overview/TopicOverviewSection.tsx

```diff
@@ -1,59 +1,67 @@
-                        if (idx === 0) {
-                            return (
-                                <Col key={element + idx}>
-                                    <Link href={`/topics/${element.toLowerCase()}`} className="text-decoration-none">
-                                        <Card style={{ height: "186px"}} className="mx-1 my-3">
-                                            <Card.Body className="m-0 p-0">
-                                                <div className="w-100 bg-primary" style={{height: "138px"}}></div>
-                                            </Card.Body>
-                                            <Card.Body className="m-2 p-1">
-                                                <p>{element.split('-').join(' ')}</p>
-                                            </Card.Body>
-                                        </Card>
-                                    </Link>
-                                </Col>
-                            )
-                        } else {
-                            return (
-                                <Col key={element + idx}>
-                                    <Link href={''} className="text-decoration-none position-relative">
-                                        <Card style={{ height: "186px"}} className="mx-1 my-3">
-                                            <Card.Body className="m-0 p-0">
-                                                <div className="w-100 bg-primary" style={{height: "138px"}}></div>
-                                            </Card.Body>
-                                            <Card.Body className="m-2 p-1">
-                                                <p>{element.split('-').join(' ')}</p>
-                                            </Card.Body>
-                                        </Card>
-                                        <div style={{zIndex: 2, top: 0, cursor: 'not-allowed'}} className="position-absolute w-100 h-100 opacity-75 bg-white"></div>
-                                    </Link>
-                                </Col>
-                            )
-                        }
-                    })}
-                </Row>
-            </Container>
-        </section>
+                <Container className="bottom-grey-border px-4 pt-1 no-max-container pb-4">
+                    <Row xl={3} lg={3} md={2} sm={2} xs={2}>
+                        {labels.map((element, idx) => {
+                            // Special if statement for conference, the 'else' will be removed for final
+                            if ((idx === 0 && section === "culture") || (idx === 2 && section === "church")) {
+                                return (
+                                    <Col key={element + idx} className="d-flex justify-content-center">
+                                        <Link href={`/topics/${element.toLowerCase()}`} className="text-decoration-none">
```

Comment:
This should be links right? once we add Japanese, the labels would be in japanese and links in english?